Update timeline of events migration (#13595)
DataDog/documentation: content/en/events/guides/migrating_to_new_events_features.md

## What is the migration timeline?

> <strong>March, 2022</strong> - New Events Explorer and analytics are accessible. Datadog begins to migrate customer dashboards and monitors not managed using the API.

Reviewer: ```suggestion
<strong>March 2022</strong> - New Events Explorer and analytics are accessible. Datadog begins to migrate customer dashboards and monitors that are not managed with APIs.
```
